### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9.

### CODE REVIEW

File: src/zope/pagetemplate/pagetemplate.py [hunk 1]
```diff
@@ -144,14 +144,26 @@ def pt_errors(self, namespace, check_macro_expansion=True):
             try:
                 self.pt_render(namespace, source=1)
             except Exception:
-                return ('Macro expansion failed', '%s: %s' % sys.exc_info()[:2])
+                return (
+                    'Macro expansion failed',
+                    '%s: %s' %
+                    sys.exc_info()[
+                        :2])
```
Comment (mgedmin) [inline src/zope/pagetemplate/pagetemplate.py:151]: That is, uh, a bit weird.

**Review by mgedmin — APPROVED** [review]
LGTM.

I've tweaked coveralls.io settings for this repo and set the failure threshold to 97% (it was 100% before, probably because we had branch coverage turned off?).

File: src/zope/pagetemplate/pagetemplate.py (hunk 1 above)
Comment (icemac) [reply src/zope/pagetemplate/pagetemplate.py]: Good catch.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**